readers.py

```diff
@@ -63,9 +61,10 @@ def get_chart_data(self):
         return chart_data
 
     def _get_space(self):
-        stat = os.statvfs(env.get_profile_path())
-        free_space = stat[statvfs.F_BSIZE] * stat[statvfs.F_BAVAIL]
-        total_space = stat[statvfs.F_BSIZE] * stat[statvfs.F_BLOCKS]
+        path = env.get_profile_path()
+        stat = os.statvfs(path)
+        free_space = stat[0] * stat[4]
+        total_space = stat[0] * stat[2]
```

Comment:
This method of accessing the statvfs result by magic numbers is not easy to understand; you should use names instead; e.g. stat.f_bsize ... try it interactively and you'll see.

Reply:
Done in 14f0084